Dockerfile

```diff
@@ -1,42 +1,42 @@
-FROM node:14-alpine
-
-RUN apk add --no-cache git \
-  python \
-  make
+# First stage
+FROM node:14.17-buster as builder
 
 ARG BUILD_NUMBER
 ARG GIT_REF
 ARG GIT_DATE
 
-# Create app directory
-WORKDIR /home/node/app
-# Install app dependencies
-# A wildcard is used to ensure both package.json AND package-lock.json are copied
-# where available (npm@5+)
-COPY package*.json ./
-RUN CYPRESS_INSTALL_BINARY=0  npm ci --no-audit
+RUN apt-get update && \
+    apt-get upgrade -y
+
+WORKDIR /app
+
+COPY . .
+
+RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit --production && npm run build
```

Comment:
I dont think there is any point in chaining these commands because the builder stage will be discarded anyway.